sdk/core/azure_core_amqp/src/fe2o3/value.rs

```diff
@@ -36,23 +36,31 @@ impl From<AmqpValue> for fe2o3_amqp_types::primitives::Symbol {
     }
 }
 
+// Number of milliseconds between the Unix epoch (1/1/1970) and year 1 CE.
+// This is the lowest value that can be represented by an AMQP timestamp.
+const CE_ZERO_MILLISECONDS: i64 = -62_135_596_800_000;
+
 impl From<fe2o3_amqp_types::primitives::Timestamp> for AmqpTimestamp {
     fn from(timestamp: fe2o3_amqp_types::primitives::Timestamp) -> Self {
         AmqpTimestamp(
-            std::time::UNIX_EPOCH
-                + std::time::Duration::from_millis(timestamp.milliseconds() as u64),
+            std::time::UNIX_EPOCH.checked_add(std::time::Duration::from_millis(
```

Comment:
I believe this will fail for `wasm32-unknown-unknown` so we'll need to condition it out or, more likely, fail with a good error message.

Reply:
FWIW, it seems to work. No idea exactly why though.

Reply:
Ah, I think I see why: we're not testing against `wasm32-unknown-unknown` in our AzDO tests it seems. Can you try that on the command line directly for now? @hallipr we'll have to make sure we testing against WASM as well.

```bash
cargo build --all-features --workspace --target wasm32-unknown-unknown
```

Reply:
Getting that command to work is a nightmare. Tons of openssl related bugs due to c_uint and c_ulong not being found.